Client/src/Pages/Maintenance/CreateMaintenanceWindow/index.jsx

```diff
@@ -45,9 +46,27 @@ const configOptionTitle = (title, description) => {
   );
 };
 
+const timeFrames = [
+  { _id: 1, name: "seconds" },
+  { _id: 2, name: "minutes" },
+  { _id: 3, name: "hours" },
+  { _id: 4, name: "days" },
```

Comment:
There's a mismatch between these values and the ones in the dropdown:

![image](https://github.com/user-attachments/assets/44dde318-8d8a-4a7c-a4f7-c335a3ca8593)

`minutes` vs `Minutes`

These can be updated to match